xbmc/music/Album.cpp

```diff
@@ -176,6 +176,28 @@ std::string CAlbum::GetGenreString() const
   return StringUtils::Join(genre, g_advancedSettings.m_musicItemSeparator);
 }
 
+const std::vector<std::string>& CAlbum::GetAlbumArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> albumartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
+  {
+    albumartists.push_back(artistCredit->GetArtist());
+  }
+  return albumartists;
```

Comment:
There is some fancy method to do all of this in one line, something like `back_inserter(artistCredits.cbegin())` or something.